README.md

```diff
@@ -0,0 +1,25 @@
+# The Dollar Effect
+
+A subscription system to sponsor a non-profit organization of your choice for $1 a month.
+
+## Getting Started
+
+To be able to make changes in the repository, you need to be added to the github of dollareffect.github.io.
+Right now, only Albert can add members. Tag him in the project text channel so he can add you.
+
+### Prerequisites
+
+1. Text Editor ([Visual Studio Code](https://code.visualstudio.com), [Sublime Text](https://www.sublimetext.com))
+2. Netlify CLI (https://www.netlify.com/products/cli/)
+
+### Local Development
+
+In the project root directory terminal, run 'netlify dev' to start local testing of your changes. The browser should open and you should be able to access it through 'http://localhost:8888'.
+
+```
+netlify dev
+```
+
+## Deployment
+
+After creating a PR and merging your changes to origin/master, it will shortly reflect on the website 'https://thedollareffect.com/'.
```

Comment:
Add the context that the dev can see the deployment status here: https://github.com/dollareffect/dollareffect.github.io/actions

Reply:
I didn't know this one. will apply this comment. Thank you.